dev/com.ibm.ws.jsf.2.3_fat/fat/src/com/ibm/ws/jsf23/fat/tests/JSF23ViewParametersTests.java

```diff
@@ -73,6 +74,7 @@ public static void tearDown() throws Exception {
      * @throws Exception
      */
     @Test
+    @SkipForRepeat(SkipForRepeat.EE9_FEATURES)
```

Comment:
Same comment we need to open an issue for investigation

Reply:
https://github.com/OpenLiberty/open-liberty/issues/13887